tests/test_basic_ops_dict.py

```diff
@@ -267,13 +267,17 @@ def test_dict_get_with_default_errors(
 
 
 def test_map_expansion() -> None:
-    c = OmegaConf.create("{a: 2, b: 10}")
+    c = OmegaConf.create({"a": 2, "b": 10})
     assert isinstance(c, DictConfig)
 
     def foo(a: int, b: int) -> int:
         return a + b
 
-    assert 12 == foo(**c)
+    a = {**c}["a"]
+    b = {**c}["b"]
+
+    # assert 12 == foo(**c) works as well, but gives lint error
+    assert 12 == foo(a, b)
```

Comment:
I am not sure if this is testing for expansion of args. This is more along the lines of `assert 12 == foo(c.a, c.b)`. Maybe it is better to silence the mypy error ?

Reply:
Yeah, that is definitely possible and seems like a better solution. Thank you for the hint. I will fix it soon...

Reply:
> I am not sure if this is testing for expansion of args.

The name of the test is `test_map_expansion`. My guess is that it *is* testing expansion of args.

Reply:
I think we should surpress the lint error.
It's more important for me that the test intention is clear than to satisfy mypy.

Reply:
The error is ignored now...